composer.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "yakamara/redaxo_ycom",
+
+    "require-dev": {
+        "redaxo/source": "^5.8",
+        "phpstan/phpstan": "0.12.52"
```

yform hier notieren, sobald es auf packagist verfügbar ist.
dann kann phpstan auch die yform klassen sehen und entsprechend gibts weniger (oder andere) fehlermeldungen von phpstan ;)

```suggestion
        "phpstan/phpstan": "0.12.52",
        "yakamara/redaxo_yform": "dev-master"
```